students/JeroenSweerts/Lesson3/mailroom.py

```diff
@@ -42,13 +45,11 @@ def report():
      After printing this report, return to the original prompt.
      At any point, the user should be able to quit their current task and return
      to the original prompt.
-     From the original prompt, the user should be able to quit the script cleanly.'''
+     From the original prompt, the user should be able to quit the script cleanly."""
     print(f'{"Donor Name":20s} {"|  Total Given":20s} {"|  Num Gifts  |":20s} {"Average Gift":20s}')
-    print(f'{"-"*80}')
+    print(f'{"-"*76}')
     for name in donors:
-        print(f'{name:20s} ${sum(donations[donors.index(name)]):20.2f}\
-        {len(donations[donors.index(name)]):12d}\
-        ${sum(donations[donors.index(name)])/len(donations[donors.index(name)]):20.2f}')
+        print(f'{name:20s} ${sum(int(i) for i in donations[donors.index(name)]):20.2f} {len(donations[donors.index(name)]):13d}${sum(int(i) for i in donations[donors.index(name)])/len(donations[donors.index(name)]):20.2f}')
```

Comment:
I'd suggest breaking these calculations out of the format string and assigning to variables like 'total_gifts', 'average_gift' etc. for readability. Also, you can use triple quoted strings for extra long strings that span multiple lines to keep under Pep8 recommended line length. See my other comment on casting type of user input so you don't need to loop through and cast donations to `int`s here.